---
title: "Analysing microbial communites with mothur and R"
author: "Kendra Maas"
date: "August 21, 2017"
output: html_document
# site: bookdown::bookdown_site
# output: bookdown::gitbook
---
# UConn MARS Workshop Aug 21-25, 2017

Microbes control the world! And with advances in molecular techniques, esp. DNA sequencing, we can explore the microbial side with ever increasing depth and breadth. But the data generated by sequencing the microbial world is bigish and weird. Most bioinformatic tools don't handle the taxonomic markers that we like to sequence very well. There are a few specialized programs for processing taxonomic sequences, we will be using mothur <http://www.mothur.org/wiki/MiSeq_SOP>. Later this week, we'll take the OTU table (species by sample matrix) into R to explore the data and learn how to make publication quality figures.

Basic Schedule. We will roughly follow this, but the exact amount of time we spend on each topic will be adjusted depending on the class.

Day|Session|Topic
------|-------|--------
Day 1|Morning|Introduction to communities
.|.|Introduction to command line
.|.|Overview of wetlab
.|Afternoon|Introduction to Basespace
.|.|Begin mothur processing
Day 2|Morning|Continue mothur processing
.|Afternoon|mothur on bigger datasets
.|.|Using HPC
Day 3|Morning|Reproducible Research
.|.|Introduction to R
.|Afternoon|Importing data into R
.|.|Subsetting, joining, data management
Day 4|Morning|Ecological statists
.|.|ggplot
.|Afternoon|Hypothesis testing
Day 5|Morning|Indicator species analysis
.|.|More complext figures
.|Afternoon|Version control?
.|.|Class requests